test/powershell/Language/Scripting/ScriptHelp.Tests.ps1

```diff
@@ -23,7 +23,7 @@ Describe 'get-help HelpFunc1' -Tags "Feature" {
         It '$x.Synopsis' { $x.Synopsis | Should -BeExactly "A relatively useless function." }
         It '$x.Description' { $x.Description[0].Text | Should -BeExactly "A description`n`n    with indented text and a blank line." }
         It '$x.alertSet.alert' { $x.alertSet.alert[0].Text | Should -BeExactly "This function is mostly harmless." }
-        It '$x.relatedLinks.navigationLink[0].uri' {  $x.relatedLinks.navigationLink[0].uri | Should -BeExactly "http://blogs.msdn.com/powershell" }
+        It '$x.relatedLinks.navigationLink[0].uri' {  $x.relatedLinks.navigationLink[0].uri | Should -BeExactly "https://blogs.msdn.com/powershell" }
```

Comment:
Seems we need revert the change.

Reply:
Test passes because line 52 is also updated.